common-custom-user-data-scripts/gradle-common-custom-user-data.gradle

```diff
@@ -319,7 +319,7 @@ String execAndGetStdout(String... args) {
     }
 }
 
-void addCustomLinkWithSearchTerms(def buildScan, String title, Map<String, String> search) {
+static void addCustomLinkWithSearchTerms(def buildScan, String title, Map<String, String> search) {
```

Comment:
How is this working without making `customValueSearchParams`  and `appendIfMissing` methods static?

Reply:
I can't say I know enough about groovy to understand this. :) I'll make them static though.